lib/irb/command/help.rb

```diff
@@ -6,27 +6,17 @@ class Help < Base
       category "Help"
       description "List all available commands. Use `help <command>` to get information about a specific command."
 
-      class << self
-        def transform_args(args)
-          # Return a string literal as is for backward compatibility
-          if args.empty? || string_literal?(args)
-            args
-          else # Otherwise, consider the input as a String for convenience
-            args.strip.dump
-          end
-        end
-      end
-
-      def execute(command_name = nil)
+      def execute(command_name)
+        # Accept string literal for backward compatibility
```

Comment:
I think this has been dropped?

```
irb(main):001> help "irb_info"
Can't find command `"irb_info"`. Please check the command name and try again.

=> nil
irb(main):002> help irb_info
Show information about IRB.
=> nil
irb(main):003> 
```

```suggestion
```

Reply:
I dropped it in this commit https://github.com/ruby/irb/pull/824/commits/47c2210555e565bd0895e563275db0e263a59258
https://github.com/ruby/irb/pull/824#discussion_r1546233617

Do you think we should revert it?

Reply:
No I meant since it's dropped, the comment isn't true anymore and can be removed?

Reply:
Thanks! removed